salesforce/my-conservation-life/force-app/main/default/lwc/utils/connection.js

```diff
@@ -1,3 +1,3 @@
-const URL = 'https://my-conservation-life.herokuapp.com/api/v1/';
+const URL = 'https://my-conservation-life-dev.herokuapp.com/api/v1/';
```

Comment:
The url should be https://my-conservation-life.herokuapp.com/api/v1/

Reply:
+1, this file should not be changed at all